ui/src/track-layout/track-layout-react-utils.tsx

```diff
@@ -199,7 +200,7 @@ export function useReferenceLineStartAndEnd(
 export function useLocationTrackStartAndEnd(
     id: LocationTrackId | undefined,
     publishType: PublishType | undefined,
-    changeTime: TimeStamp,
+    changeTime: TimeStamp = getChangeTimes().layoutLocationTrack,
```

Comment:
Tarvitsiko tätä? Olen käyttänyt näitä hätävarana koska joskus ei ole järkeä bindata changetimeä läpi komponentin, mutta splittauksessa (ja infoboxissa yleensä) se lienee järkevää, jotta varmistetaan että tieto päivittyy. Täällä defaultattu changetime "toimii" mutta ei aiheuta komponentin automaattista päivittymistä.